data/sles4sap/qe_sap_deployment/qesap_azure_fencing_spn.yaml

```diff
@@ -3,7 +3,8 @@
 # Copyright SUSE LLC
 # SPDX-License-Identifier: FSFAP
 # Maintainer: QE-SAP <qe-sap@suse.de>
-# Summary: Generic yaml template for use with qe-sap-deployment project: https://github.com/SUSE/qe-sap-deployment
+# Summary: conf.yaml template for qe-sap-deployment project: https://github.com/SUSE/qe-sap-deployment
+#          Deploy on Azure, two node HANA cluster with cloud native SPN fencing.
```

Comment:
I don't see a reference to `SPN` in the YAML, and IIRC it defaults to MSI, right?